.github/workflows/cronjobs.yaml

```diff
@@ -30,3 +30,53 @@ jobs:
         if: ${{ env.NUM_OPEN_PRS == 0 }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  check-image-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Retrieve LATEST_SHA
+        run: |
+          LATEST_SHA=$(skopeo inspect --raw docker://registry.access.redhat.com/ubi9/go-toolset:latest | jq -r '.manifests[] | select(.platform.architecture == "amd64") .digest')
+          echo "LATEST_SHA=$LATEST_SHA" >> $GITHUB_ENV
+
+      - name: Retrieve CURRENT_SHA
+        working-directory: redhat/overlays
+        run: |
+          CURRENT_SHA=$(grep FROM Dockerfile | awk '{print $2}' | awk -F '@' '{print $2; exit}')
```

Comment:
This only updates `Dockerfile`, not `Dockerfile.*`. That's probably fine for this first iteration, but I'd like to improve this action so that it can work with all Dockerfiles in a repo. E.g. turn this one-liner into a small script in `redhat/hack/update-shas.sh` that can take a filename to process, and an image to update (this only updates go-toolset, but we also need updates to ubi-minimal), then call that script from here for every `Dockerfile.*` in `redhat/overlays`.

```
source ../hack/update-shas.sh
for N in go-toolset ubi-minimal ; do
  for F in Dockerfile.* ; do
    update_shas $F $N;
  done
done
```
But I'm not sure off the top of my head how you would incorporate that with the structure of the rest of this action.